src/include/kernel/mem/vmm.h

```diff
@@ -2,7 +2,9 @@
 #define __VMM_H
 
 #include <stddef.h>
+#include <bitmap.h>
 
+#define NONE 0
 #define PRESENT 0b1
 #define WRITE_ENABLE 2
```

Comment:
nitpick: why is `PRESENT` defined with a bit prefix, and WRITE/USER use integers.